### ISSUE TITLE
Trophyroom blocks

### ISSUE THREAD

**Issue by Syxes** [Carnivores-mc-mod/Carnivores-mc-mod#23]
Can you add the rotate when placed thing we talked about to all of them?
Make them like iron blocks.

Thanks for your hard work.

**Comment by Syxes** [comment]
https://www.dropbox.com/s/p1k3ics2ythu4px/Trophy.rar?dl=0
Here is the link for some blocks, which need coded.
You can start with the none custom model ones.
No hurry, just when you have time.

**Comment by Caske2000** [comment]
Are these cubical or like a cylinder?

**Comment by Syxes** [comment]
they are rectangular

**Comment by Caske2000** [comment]
Do I code them just as regular sized blocks? Are they a multiblock?

**Comment by Syxes** [comment]
There are 5 kinds: high*length*width in blocks
corner: high: 0,5*1*1
platform: 1*3*3
stair: 0,25*1*1
wall: 1*0,5*1

**Comment by Caske2000** [comment]
Can you also pass the details of the blocks please, hardness, steppsound, etc?

**Comment by Syxes** [comment]
Like iron block and stepsound also ironblock
If I missed anything let me know

**Comment by Caske2000** [comment]
Darthvader, do you know how to change the facing of a block when placing it?

**Comment by Caske2000** [comment]
The tileblocks' code is done, I will upload it tomorrow, after a doublecheck

**Comment by Syxes** [comment]
Great

**Comment by darthvader45** [comment]
Caske, for the rotation, check this topic, it should have something similar
to what you're looking for.

http://www.minecraftforge.net/forum/index.php?topic=13978.0

On Sat, Jan 3, 2015 at 4:18 PM, Syxes <notifications@github.com> wrote:

> Great
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Carnivores-mc-mod/Carnivores-mc-mod/issues/23#issuecomment-68609964>
> .
>



-- 
Golden rule in Minecraft: Never dig straight down.

**Comment by Caske2000** [comment]
Is it necessary to make it a TE?

**Comment by Syxes** [comment]
Does the facing work?

**Comment by Caske2000** [comment]
Still not... The metadata, I can fix, but I m having problems with the Glrotatef

**Comment by Syxes** [comment]
The what?

**Comment by darthvader45** [comment]
The rotate code, Syxes.  GlRotateF rotates the block.

On Sun, Jan 4, 2015 at 11:23 AM, Syxes <notifications@github.com> wrote:

> The what?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Carnivores-mc-mod/Carnivores-mc-mod/issues/23#issuecomment-68638467>
> .
>



-- 
Golden rule in Minecraft: Never dig straight down.

**Comment by Caske2000** [comment]
Yes, when you add a block to the world, then you give it a metadata value that corresponds to the rotation of the player when you placed the block (2,3,4,5). Then in the renderclass of the block you need to rotate(GL11.glRotatef(angle, axisX, axisY, axisZ)) the texture of the block, but I am having problems doing that. (The metadata part is working.)

**Comment by Syxes** [comment]
May we can ask Ri5ux form the AvP. mod?

**Comment by Caske2000** [comment]
Yes, he seems like an experienced coder

**Comment by Syxes** [comment]
Ok let's try

**Comment by darthvader45** [comment]
Agreed.

On Sun, Jan 4, 2015 at 3:13 PM, Syxes <notifications@github.com> wrote:

> Ok let's try
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Carnivores-mc-mod/Carnivores-mc-mod/issues/23#issuecomment-68646990>
> .
>



-- 
Golden rule in Minecraft: Never dig straight down.

**Comment by Syxes** [comment]
OK one of you should send him a pm on mcforum.

**Comment by Syxes** [comment]
Ok asked him, now wait for reply.
